app/models.py

```diff
@@ -0,0 +1,61 @@
+from . import db
+import datetime
+
+
```

Comment:
This can probably be removed now as serialise and deserialise will be done in the schema class.